server/views/components/article-body/index.njk

```diff
@@ -25,6 +25,8 @@
         {% component 'tweet', { tweet: bodyPart.value } %}
       {% elif bodyPart.type === 'instagramEmbed' %}
         {% component 'instagram-embed', { instagramEmbed: bodyPart.value } %}
+      {% elif bodyPart.type === 'blockquote' %}
+        {% component 'blockquote', { body: bodyPart.value.body | safe, footer: bodyPart.value.cite | safe, modifiers: ['small'] } %}
```

Comment:
`--small` = `blockquote`.
The other ones we don't use as yet - although I think we agreed that the normal / large one looked like a pullquote.